packages/light-client/src/spec/utils.ts

```diff
@@ -105,6 +105,14 @@ export function upgradeLightClientHeader(
 
       // Break if no further upgradation is required else fall through
       if (ForkSeq[targetFork] <= ForkSeq.deneb) break;
+
+    // eslint-disable-next-line no-fallthrough
+    case ForkName.eip6110:
+      (upgradedHeader as eip6110.LightClientHeader).execution.depositReceiptsRoot =
+        ssz.eip6110.LightClientHeader.fields.execution.fields.depositReceiptsRoot.defaultValue();
+
+      // Break if no further upgradation is required else fall through
```

Comment:
(aware this was already here 🤣 )
```suggestion
      // Break if no further upgrades is required else fall through
```

Reply:
> (aware this was already here 🤣 )

Lol